single_asset/README.md

```diff
@@ -9,8 +9,9 @@ contract interfaces defined in
 [LIGO](https://ligolang.org/), smart-contract language for Tezos
 * [`impl`](impl/) folder - reference implementation of the multi-asset contract,
 test helper contracts and code.
-* [`out`](out/) folder - multi-assert contract and helper contract compiled into
+* [`out`](out/) folder - multi-asset contract and helper contract compiled into
 Michelson
+<!-- OOC why is the output checked in the repo? -->
```

Comment:
this is the one compiled with current version of LIGO which can be originated on blockchain w/o recompiling the source. Helps with uncertainty and instability of LIGO output